app/helpers/checkout_helper.rb

```diff
@@ -40,6 +40,21 @@ def checkout_adjustments_for(order, opts = {})
       )
     end
 
+    tax_rate_adjustments, adjustments = adjustments.partition { |a|
+      a.originator_type == "Spree::TaxRate" && a.adjustable_type == "Spree::Adjustment"
```

Comment:
This will only match taxes on fees and not taxes on line items or shipments. Is that intended?

If you want it on all tax adjustments you can do
```diff
-- a.originator_type == "Spree::TaxRate" && a.adjustable_type == "Spree::Adjustment"
++ a.originator_type == "Spree::TaxRate"
```

Reply:
Yep, I think that's right - based on the issue, it seemed like the problem was specifically with taxes on fees showing up individually. But it makes more sense that we would want to group all adjustments related to a particular tax together. b5881c2ec88ed4523fc42673ef8a289afd27cab3